stl/inc/yvals.h

```diff
@@ -7,6 +7,12 @@
 #ifndef _YVALS
 #define _YVALS
 #include <yvals_core.h>
+
+#ifdef _ENFORCE_ONLY_CORE_HEADERS
+static_assert(false, "error STL2001: You tried to include a non-core C++ standard header file, although the mode is "
+                     "set in which only core C++ standard header files are allowed");

```

Comment:
Maybe rephrase it a bit
```suggestion
static_assert(false, "error STL2001: You tried to include a non-core C++ standard header file. "
                     "However, with `_ENFORCE_ONLY_CORE_HEADERS` only core C++ standard header files are allowed");
```

Reply:
I am not really married to although / however, but I believe it is important to mention the flag that has been defined